### PR TITLE
Some bug fixes and bash completion

### PR DESCRIPTION
**Pull request by cardosolucas**
- grpchost/grpcport related lines that I forgot last patch changed;
- Jupyter-lab security flag removed in CLI;
- Bash completion added.

### CODE REVIEW

**Comment by cardosolucas** [comment]
I will make smaller but frequent PRs now. It is easier to review this way!

File: marvin-cli/marvin_cli/management/engine.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def import_project(file, dest):
     type=click.Choice(['all', 'acquisitor', 'tpreparator', 'trainer', 'evaluator', 'ppreparator', 'predictor', 'feedback']),
     help='Marvin engine action name')
 @click.option('--profiling', '-p', default=False, is_flag=True, help='Deterministic profiling of user code.')
-def dryrun(host, port, action, profiling):
+def dryrun(grpchost, grpcport, action, profiling):
     rc = RemoteCalls(host, port)
```
Comment (Wei-1) [inline marvin-cli/marvin_cli/management/engine.py]: This line should have the same modification?

File: marvin-cli/marvin_cli/management/test.py [hunk 2]
```diff
@@ -23,32 +23,32 @@ def cli():
...
-@click.option('--port', '-p', prompt='gRPC port', help='gRPC Port', default='50057')
+@click.option('--grpchost', '-gh', prompt='gRPC host', help='gRPC Host Address', default='localhost')
+@click.option('--grpcport', '-gp', prompt='gRPC port', help='gRPC Port', default='50057')
 @click.option('--cov/--no-cov', default=True)
 @click.option('--no-capture', is_flag=True)
 @click.option('--pdb', is_flag=True)
 @click.argument('args', default='')
-def test(host, port, cov, no_capture, pdb, args):
+def test(grpchost, grpcport, cov, no_capture, pdb, args):
     rc = RemoteCalls(host, port)
```
Comment (Wei-1) [inline marvin-cli/marvin_cli/management/test.py]: Should change with the above line?

File: marvin-cli/marvin_cli/management/test.py [hunk 3]
```diff
@@ -23,32 +23,32 @@ def cli():
...
 
 @cli.command('test-tox', help='Run tests using Tox environment.')
-@click.option('--host', '-h', prompt='gRPC host', help='gRPC Host Address', default='localhost')
-@click.option('--port', '-p', prompt='gRPC port', help='gRPC Port', default='50057')
+@click.option('--grpchost', '-gh', prompt='gRPC host', help='gRPC Host Address', default='localhost')
+@click.option('--grpcport', '-gp', prompt='gRPC port', help='gRPC Port', default='50057')
 @click.argument('args', default='--current-env')
-def tox(host, port, args):
+def tox(grpchost, grpcport, args):
     rc = RemoteCalls(host, port)
```
Comment (Wei-1) [inline marvin-cli/marvin_cli/management/test.py]: Should change with the above line?

File: marvin-cli/marvin_cli/management/test.py [hunk 4]
```diff
@@ -23,32 +23,32 @@ def cli():
...
+@click.option('--grpchost', '-gh', prompt='gRPC host', help='gRPC Host Address', default='localhost')
+@click.option('--grpcport', '-gp', prompt='gRPC port', help='gRPC Port', default='50057')
 @click.option('--cov/--no-cov', default=False)
 @click.option('--no-capture', is_flag=True)
 @click.option('--pdb', is_flag=True)
 @click.option('--partial', is_flag=True)
 @click.argument('args', default='')
-def tdd(host, port, cov, no_capture, pdb, partial, args):
+def tdd(grpchost, grpcport, cov, no_capture, pdb, partial, args):
     rc = RemoteCalls(host, port)
```
Comment (Wei-1) [inline marvin-cli/marvin_cli/management/test.py]: Should change with the above line?

File: marvin-cli/marvin_cli/management/engine.py (hunk 1 above)
Comment (cardosolucas) [reply marvin-cli/marvin_cli/management/engine.py]: Done!

File: marvin-cli/marvin_cli/management/test.py (hunk 2 above)
Comment (cardosolucas) [reply marvin-cli/marvin_cli/management/test.py]: Done!

File: marvin-cli/marvin_cli/management/test.py (hunk 3 above)
Comment (cardosolucas) [reply marvin-cli/marvin_cli/management/test.py]: Done!

File: marvin-cli/marvin_cli/management/test.py (hunk 4 above)
Comment (cardosolucas) [reply marvin-cli/marvin_cli/management/test.py]: Done!

**Comment by cardosolucas** [comment]
Thanks for review! I fixed all the stuff you pointed.

**Comment by Wei-1** [comment]
LGTM, anyone else has other idea? Or I will merge it before 2020-10-20.

**Pull request merged**